### ISSUE TITLE
Caddy restart loop

### ISSUE THREAD

**Issue by Markcley** [api-platform/api-platform#1881]
Hello,

I'm trying to start my project (based on 2.6.3) after a docker system prune -a --volume and Caddy is not starting properly.

I tried to download the last version of API Platform (2.6.4) and i have the same issue. My colleagues face the same issue with our project or with API Platform 2.6.4.

I also found someone with the same issue on stackoverflow : https://stackoverflow.com/questions/67153080/caddy-not-working-in-api-platfrom-2-6-4-distribution-panic-proto-file-pb-pr

You can reproduce this by doing the following command lines
```
docker-compose down -v
docker system prune -a --volume
docker-compose pull
docker-compose up -d
```

Docker compose logs :
```
docker-compose logs caddy 
Attaching to api-platform-264_caddy_1
caddy_1     | panic: proto: file "pb.proto" is already registered
caddy_1     | See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict
caddy_1     | 
caddy_1     | 
caddy_1     | goroutine 1 [running]:
caddy_1     | google.golang.org/protobuf/reflect/protoregistry.glob..func1(0x1c6a100, 0xc00045a540, 0x1c35ac0, 0xc000325580, 0xc00045a540)
caddy_1     |   google.golang.org/protobuf@v1.26.0/reflect/protoregistry/registry.go:54 +0x25f
caddy_1     | google.golang.org/protobuf/reflect/protoregistry.(*Files).RegisterFile(0xc0001940c0, 0x1c73120, 0xc00045a540, 0x0, 0x0)
caddy_1     |   google.golang.org/protobuf@v1.26.0/reflect/protoregistry/registry.go:127 +0xb72
caddy_1     | google.golang.org/protobuf/internal/filedesc.Builder.Build(0x0, 0x0, 0xc000459200, 0x47a, 0x600, 0x800000003, 0x0, 0x1c49200, 0xc000186150, 0x1c52700, ...)
caddy_1     |   google.golang.org/protobuf@v1.26.0/internal/filedesc/build.go:113 +0x1aa
caddy_1     | github.com/golang/protobuf/proto.RegisterFile(0x197dac2, 0x8, 0x26468e0, 0x2b3, 0x2b3)
caddy_1     |   github.com/golang/protobuf@v1.5.2/proto/registry.go:48 +0x147
caddy_1     | github.com/dgraph-io/badger/v2/pb.init.1()
caddy_1     |   github.com/dgraph-io/badger/v2@v2.2007.2/pb/pb.pb.go:650 +0x5a
caddy_1     | panic: proto: file "pb.proto" is already registered
caddy_1     | See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict
caddy_1     | 
caddy_1     | 
caddy_1     | goroutine 1 [running]:
caddy_1     | google.golang.org/protobuf/reflect/protoregistry.glob..func1(0x1c6a100, 0xc0002d4540, 0x1c35ac0, 0xc0002f40b0, 0xc0002d4540)
caddy_1     |   google.golang.org/protobuf@v1.26.0/reflect/protoregistry/registry.go:54 +0x25f
caddy_1     | google.golang.org/protobuf/reflect/protoregistry.(*Files).RegisterFile(0xc000184150, 0x1c73120, 0xc0002d4540, 0x0, 0x0)
caddy_1     |   google.golang.org/protobuf@v1.26.0/reflect/protoregistry/registry.go:127 +0xb72
caddy_1     | google.golang.org/protobuf/internal/filedesc.Builder.Build(0x0, 0x0, 0xc0002d3200, 0x47a, 0x600, 0x800000003, 0x0, 0x1c49200, 0xc000180480, 0x1c52700, ...)
caddy_1     |   google.golang.org/protobuf@v1.26.0/internal/filedesc/build.go:113 +0x1aa
caddy_1     | github.com/golang/protobuf/proto.RegisterFile(0x197dac2, 0x8, 0x26468e0, 0x2b3, 0x2b3)
caddy_1     |   github.com/golang/protobuf@v1.5.2/proto/registry.go:48 +0x147
caddy_1     | github.com/dgraph-io/badger/v2/pb.init.1()
caddy_1     |   github.com/dgraph-io/badger/v2@v2.2007.2/pb/pb.pb.go:650 +0x5a
caddy_1     | panic: proto: file "pb.proto" is already registered
caddy_1     | See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict
caddy_1     | 
caddy_1     | 
caddy_1     | goroutine 1 [running]:
caddy_1     | google.golang.org/protobuf/reflect/protoregistry.glob..func1(0x1c6a100, 0xc0002d4540, 0x1c35ac0, 0xc0002f20a0, 0xc0002d4540)
caddy_1     |   google.golang.org/protobuf@v1.26.0/reflect/protoregistry/registry.go:54 +0x25f
caddy_1     | google.golang.org/protobuf/reflect/protoregistry.(*Files).RegisterFile(0xc000184110, 0x1c73120, 0xc0002d4540, 0x0, 0x0)
caddy_1     |   google.golang.org/protobuf@v1.26.0/reflect/protoregistry/registry.go:127 +0xb72
caddy_1     | google.golang.org/protobuf/internal/filedesc.Builder.Build(0x0, 0x0, 0xc0002d3200, 0x47a, 0x600, 0x800000003, 0x0, 0x1c49200, 0xc000180390, 0x1c52700, ...)
caddy_1     |   google.golang.org/protobuf@v1.26.0/internal/filedesc/build.go:113 +0x1aa
caddy_1     | github.com/golang/protobuf/proto.RegisterFile(0x197dac2, 0x8, 0x26468e0, 0x2b3, 0x2b3)
caddy_1     |   github.com/golang/protobuf@v1.5.2/proto/registry.go:48 +0x147
caddy_1     | github.com/dgraph-io/badger/v2/pb.init.1()
caddy_1     |   github.com/dgraph-io/badger/v2@v2.2007.2/pb/pb.pb.go:650 +0x5a
caddy_1     | panic: proto: file "pb.proto" is already registered
caddy_1     | See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

...
```

**Comment by icfmp** [comment]
Hi,

Same here, after a fresh install of Ubuntu on my computer i'm trying to up API Platform and Caddy is restarting, and restarting, and restarting, ...

I also tried with the latest version of API Platform and I'm facing the same issue.

I'm totally stuck and I can't work anymore ... Please help :cry:

**Comment by dunglas** [comment]
I'm on it. It's because of a "fix" I pushed in Mercure. In the meantime you can disable Mercure.

**Comment by dunglas** [comment]
This should now be (temporarily) "fixed" by dunglas/mercure@c9763e6.
